src/content/docs/tutorial-reporting-data/capture-the-right-data.mdx

```diff
@@ -0,0 +1,121 @@
+
+## Understand the benefits [#understand-benefits]
+
+Involving developers in this process has the following benefits:
+
+1. Improved troubleshooting:
+  * Good telemetry naming gives operations staff a common language to use with developers during incidents, reducing the time to triage and fix problems.
+  * More precise and contextually relevant telemetry from your service allows for more accurate detection of faults that you can take action on.
+
+2. Better informed development decisions by:
+  * Detecting areas of volatility or unexpected behavior and addressing them.
+  * Understanding what dependencies in your code lack redundancy and taking measures to improve the service.
+  * Appreciating how end-users are employing your software. You can better understand where improvements will have the biggest impact.
+
+## Use Key performance indicators [#key-perf-indicators]
+
+It's important know a few simple KPIs to track the ongoing improvements in your software delivery and operations programs. Here are two main types of KPIs to consider as you improve instrumentation:
+
+* **Business KPIs** are aligned to your overall program objectives and should be consistently measured to demonstrate ongoing improvement for each service. Business KPIs include: 
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-service-quality"
+    title="Service quality"
+  >
+A metric is required to define how well your service is operating. This will depend upon the needs of your organization and the constraints of the services being monitored.
+
+**Goal:** Improved service quality attainment score over time.
+
+**Best practices:**
+* Create a graphical representation as a trend of service quality achievement for defined periods (monthly and/or quarterly).
+* Service Apdex can provide an effective service-specific quality score. (See [Apdex: Measure user satisfaction](/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction/).)
+* A well defined service level management (SLM) approach using SLIs that describe the level of expected operation for service boundaries can be a useful way to establish a single measurement of quality.
+  </Collapser>
+
+  <Collapser
+    id="kpi-release-frequency"
+    title="Release frequency"
+  >
+"Release frequency" refers to the number of releases for a given service. This indicates the release velocity of the software delivery organization. 
+
+**Goal:** Consistency of attainment with consistent or improving service quality indicator.
+
+**Best practices:**
+* Derive data from deployment markers or other events sent to New Relic.
+* Measure directly from code or project management tools such as Jira, BitBucket, GitHub.
+* Consider implementing a collection mechanism to capture release events and store them directly in New Relic. See [New Relic CICD innovation](https://github.com/newrelic-experimental/nr1-innovation-cicd/tree/main/Webhooks) for example JIRA and BitBucket configs.
```

Comment:
I know `jira` capitalization is weird so I wanna confirm all caps is correct here

Reply:
I've seen it both ways but I wanted to see if anyone said anything one way or the other. I have gone with your suggestion.